### PR TITLE
[DOC] How to connect MySQL with SQLFlow

### PR DESCRIPTION
**Pull request by tonyyang-svail**

### CODE REVIEW

File: doc/run_with_mysql.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+mysql://root:root@tcp(127.0.0.1:3306)/iris?maxAllowedPacket=0
+```
+Using the `datasource`, you may launch an all-in-one Docker container by running:  
+```bash
+> docker run --rm -p 8888:8888 sqlflow/sqlflow bash -c \
+"sqlflowserver --datasource='mysql://root:root@tcp(127.0.0.1:3306)/iris?maxAllowedPacket=0' &
+SQLFLOW_SERVER=localhost:50051 jupyter notebook --ip=0.0.0.0 --port=8888 --allow-root --NotebookApp.token=''"
+```
+
+Open [localhost:8888](localhost:8888) through a web browser, and you will find there are many SQLFlow tutorials, e.g. `iris-dnn.ipynb`. Please follow the tutorials and substitute the data for your use.
```
Comment (weiguoz) [inline doc/run_with_mysql.md]: It looks \[localhost:8888](localhost:8888) won't be rendered as a link. So, let's just write `localhost:8888` here

File: doc/run_with_mysql.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+## Create a MySQL server locally for testing
+
+Our official Docker image has installed `mysql` server. We can start it by `docker run --rm -it sqlflow/sqlflow bash`, then type `service mysql start` in the console. We should be able to see
```
Comment (weiguoz) [inline doc/run_with_mysql.md]: installed `mysql` => installed a `mysql` ?

File: doc/run_with_mysql.md (hunk 1 above)
Comment (tonyyang-svail) [reply doc/run_with_mysql.md]: Sure, thanks.

File: doc/run_with_mysql.md (hunk 2 above)
Comment (tonyyang-svail) [reply doc/run_with_mysql.md]: Sure, thanks.

**Review by weiguoz — APPROVED** [review]
LGTM

File: doc/run_with_mysql.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# How to connect MySQL with SQLFlow
+
+This tutorial explains how to connect SQLFlow with [MySQL](https://en.wikipedia.org/wiki/MySQL).
+
+## Connect Existing MySQL server
```
Comment (typhoonzero) [inline doc/run_with_mysql.md:5]: should also update https://github.com/sql-machine-learning/sqlflow/blob/develop/doc/run/docker.md#mysql ?

**Pull request merged**